tm2/pkg/commands/command.go

```diff
@@ -105,6 +106,16 @@ func (c *Command) AddSubCommands(cmds ...*Command) {
 	}
 }
 
+// Main is a helper function for the command entry. It wraps the ParseAndRun and handling the flag.ErrHelp error so that every command with -h or --help won't show an error message: "error parsing commandline arguments: flag: help requested"
+func (c *Command) Main(ctx context.Context, args []string) {
```

Comment:
We already have a helper `ParseAndRun()` (it just calls `Parse()` and `Run()`)? Do we really need another wrapper (i.e `func (c *Command) Main(...)`)?

IMO, we should directly modify `ParseAndRun()` to handle this condition.

Reply:
> We already have a helper `ParseAndRun()` (it just calls `Parse()` and `Run()`)? Do we really need another wrapper (i.e `func (c *Command) Main(...)`)?
> 
> IMO, we should directly modify `ParseAndRun()` to handle this condition.

Hi sir, I'm not sure if this would affect others that use `ParseAndRun()`, which was expected to return an error instead of exiting the process. it would be better compatibility to use a new function to wrap the `ParseAndRun()` IMO

Reply:
I think it's fine to leave `ParseAndRun` as is, as it's a lib method (from `ffcli`), not something we created.

I like the error wrapping in `Main`, but I would suggest you rename this method to something more descriptive, such as `Run` (or something similar), given this is a member method.

@gfanton what do you think?

Reply:
> I think it's fine to leave `ParseAndRun` as is, as it's a lib method (from `ffcli`), not something we created.
> 
> I like the error wrapping in `Main`, but I would suggest you rename this method to something more descriptive, such as `Run` (or something similar), given this is a member method.
> 
> @gfanton what do you think?

How about changing `Main` to `Exec`: `func (c *Command) Exec(ctx context.Context, args []string)`, as there already has `func (c *Command) Run(ctx context.Context) (err error)` for `Command`
![image](https://github.com/gnolang/gno/assets/25278203/65988ea1-3241-437b-83a1-2dfbc30136cb)

Reply:
@Halimao 
Sounds good 😎